src/js/me-i18n.js

```diff
@@ -112,34 +112,101 @@
 		},
 
 		/**
-		 * Translate a string to a specified language
+		 * Translate a string to a specified language, including optionally a number to pluralize translation
 		 *
 		 * @param {String} input
+		 * @param {Number} pluralReplacement
```

Comment:
Usually, arguments provided in a translation methods are understood by developpers as values being used in the translation message (exatly the simple use %1 was providing), indicating in the variable name that it is a number and it may also provides additionnal formating such as management of plural is a little bit confusing I think.
Maybe use a simpler and generic name like `msgParams`, `params`, `values`, `args` ?

Another possibility would be to omit this explicit variable declaration and use the `arguments` array (shifted by one to remove the input), like the original implementation of format method. It would allow any number of arguments and would not imply their type and feature (which are provided by filters). Of course this is more complex and not needed given the current use cases in mejs. So supporting only one message parameter is fine for me.